src/Nethermind/Nethermind.JsonRpc.Benchmark/EthModuleBenchmarks.cs

```diff
@@ -66,7 +66,7 @@ public void GlobalSetup()
             stateProvider.Commit(spec);
             stateProvider.CommitTree(0);
 
-            WorldStateManager stateManager = new WorldStateManager(stateProvider, trieStore, dbProvider, LimboLogs.Instance);
+            OverridableWorldStateManager stateManager = new(dbProvider, trieStore.AsReadOnly(), LimboLogs.Instance);
```

Comment:
There's actually what seems like a bug, present even in `master` branch - balance set a few lines above is not returned in benchmarked `eth_getBalance` call, meaning setup is probably incorrect.